spring-integration-file/src/main/java/org/springframework/integration/file/dsl/Files.java

```diff
@@ -226,4 +233,37 @@ public static FileToByteArrayTransformer toByteArrayTransformer(boolean deleteFi
 		return fileToByteArrayTransformer;
 	}
 
+	/**
+	 * Create a {@link FileToStringTransformer} with {@link Charset#defaultCharset()}.
+	 * @return the transformer.
+	 * @since 5.2
+	 */
+	public static FileToStringTransformer fileToString() {
```

Comment:
Wait (and sorry), looks like we have there already here:
```
public static FileToStringTransformer toStringTransformer(String charset) {
...
public static FileToByteArrayTransformer toByteArrayTransformer() {
```

😢

Reply:
Doh